### PR TITLE
Custom add homepage

### PR DESCRIPTION
**Pull request by jennifer-choo**
ticket: https://trello.com/c/VbElNjAo
Screenshot 2022-10-06 at 15 11 32

### CODE REVIEW

File: controller/JobsController.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ var router = express.Router();
 var jobdata = require('../data/jobdata.js')
 
 router.get('/', async (req, res) => {     
-  res.redirect('job-roles') 
+  res.redirect('index')
```
Comment (Kirsten1822) [inline controller/JobsController.js:6]: Just some weird spacing in the JobsController.js file - line 27/28

File: public/css/style.css [hunk 2]
```diff
@@ -1,6 +1,8 @@
+
```
Comment (Kirsten1822) [inline public/css/style.css]: More weird spacing in the style.css file - specifically lines 1 and 110
Also a few spaces needed after : in line 39 for example, so just have a look through the file again to fix any of those
Also needs a new line at the end of file

File: test/unit/Roles/JobRolesTests.js [hunk 3]
```diff
@@ -7,7 +7,8 @@ const URL = "http://localhost:8080/api/job-roles"
 
 const jobRoles = {
   job_description: "Builds the software from requirements set by clients",
-  name: "Software Engineer"
+  name: "Software Engineer",
+  band_name: "Apprentice"
```
Comment (Kirsten1822) [inline test/unit/Roles/JobRolesTests.js]: Again, more weird spacing in the JobRolesTests.js file - specifically line 18/19/20 and line 63/64

File: views/index.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{% set title = "Kainos Careers" %}
+{% extends "template.html" %}
+{% block content %}
+    <br><br>
+
+    <div class = "homeContainer"><div class="homeTextContainer"><h1>Kainos Careers</h1><h2>Here at Kainos, we have a variety of jobs at different levels.</h2> 
+        <br> <input type="button" class="button" onclick="location.href='http://localhost:3000/job-roles';" value="View More Roles" />
+        </div><img src="https://www.kainos.com/globalassets/images/3_size-3-content-modules-cards-desktop-insights/level-1-end-user/level-1-size-3-51.jpg?mode=crop&format=webp&width=1104&heightratio=0.625&quality=80">
+    </div>
+{% endblock %}
```
Comment (Kirsten1822) [inline views/index.html]: Needs a new line at the end of file

**Review by Kirsten1822 — CHANGES_REQUESTED** [review]

**Review by Kirsten1822 — APPROVED** [review]

**Review by seaninbreslin — APPROVED** [review]

**Review by Kirsten1822 — APPROVED** [review]

File: views/index.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{% set title = "Kainos Careers" %}
+{% extends "template.html" %}
+{% block pageTitle %}
+  <h1>Homepage</h1>
```
Comment (boydstepnew) [inline views/index.html]: lets remove this title

File: views/template.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    <img src="https://upload.wikimedia.org/wikipedia/en/3/3e/Kainos_logo.jpg">
+<ul>
+    <li><a href="http://localhost:3000/">Home</a></li>
+    <li><a href="http://localhost:3000/job-roles">Job Roles</a></li>
+</ul>
+</div>
+{% endblock %}
+
+{% block pageTitle %}
+  <h1>Kainos Careers</h1>
```
Comment (boydstepnew) [inline views/template.html]: Should this be in the template?

**Review by Kirsten1822 — APPROVED** [review]

File: views/template.html (hunk 6 above)
Comment (Jadevious) [reply views/template.html]: Yep, the pageTitle in template just provides a default page title if no other is specified when another page inherits from the template

**Review by boydstepnew — APPROVED** [review]

**Pull request merged**